garak/command.py

```diff
@@ -6,6 +6,8 @@
 import logging
 import json
 
+import garak.cli
```

Comment:
This creates something of a circular dependency, `cli.main()` imports `garak.command`.  The `hint` method should likely be factored out of cli.

Reply:
Agree, it's not perfect here. I would like to use it more places. It's in CLI because it's used a bit here, it came from here, and because it prints (i.e. does CLI stuff) - a presentation/logic separation isn't near on the roadmap. Where do you think could be an OK home for it?

Reply:
> I would like to use it more places.
Some info on `where` else it would be used can guide ideas for location.

For the short term I can see placing it in `command` since we already `know` we want to extract `print` statements  in the future. Avoiding the `cycle` is more pressing that not having one more `print` statement a class that already has them.

Reply:
yeah, agree. thanks for the catch. pushed